public/app/features/alerting/unified/api/prometheus.ts

```diff
@@ -14,6 +14,9 @@ export async function fetchRules(dataSourceName: string): Promise<RuleNamespace[
 
   const nsMap: { [key: string]: RuleNamespace } = {};
   response.data.data.groups.forEach((group) => {
+    group.rules.forEach((rule) => {
+      rule.query = rule.query || ''; // @TODO temp fix, backend response ism issing query. remove once it's there
```

Comment:
nit: do we have any epic issue where we track there things?

Reply:
I added some issues for the TODOs to https://github.com/grafana/alerting-squad/projects/2
Been a bit neglectful of maintaing this, will try harder